bin/list.js

```diff
@@ -0,0 +1,28 @@
+var parents = require('parents');
+
+var findPackageJSON = require('../lib/find-package-json.js');
+var findUsedModules = require('../lib/find-used-modules');
+
+module.exports = function(filePath, callback) {
+
+  var data = {
+    folders: parents(filePath),
+    idx: 0,
+    filePath: filePath
+  }
+
+  findPackageJSON(data)
+    .then(findUsedModules)
+    .then(function(data) {
+      data.modules.forEach(m => {
+        var dep = data.packageJSON.dependencies[m];
+        var dev = data.packageJSON.devDependencies[m];
+        console.log(m+'@'+(dep || dev));
+      });
+    })
+    .catch(err => {
+      throw err;
```

Comment:
Is there good reason to catch & throw, or should you just let the caller `.catch()`?

Also seeing that this function accepts a callback that doesn't get fired.